automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoApiClient.scala

```diff
@@ -43,7 +43,7 @@ object LeonardoApiClient {
   val client: Resource[IO, Client[IO]] = for {
     blockingEc <- ExecutionContexts.cachedThreadPool[IO]
     client <- blaze.BlazeClientBuilder[IO](blockingEc).resource
-  } yield Logger[IO](logHeaders = false, logBody = true)(client)
+  } yield Logger[IO](logHeaders = true, logBody = true)(client)
```

Comment:
Does this log the token? Might be good to redact it if we can.

Reply:
it is redacted automatically! I think it might be new thing in newer library cuz I remember I didn't enable this previously because of that...

This is really nice now since we can see `traceId` from these requests